build/verify-features.ts

```diff
@@ -29,12 +29,29 @@ function findCssFileError(filename: string): string | void {
 	}
 }
 
+function findGqlFileError(filename: string): string | void {
+	const basename = filename.replace('.gql', '');
+	const featureId = importedFeatures.find(featureId => basename.startsWith(featureId));
+	if (!featureId) {
+		return `ERR: ${filename} doesn’t match any existing features. The filename should match the feature that uses it.`;
+	}
+
+	const correspondingTsxFile = `source/features/${featureId}.tsx`;
+	if (!readFileSync(correspondingTsxFile).includes(`from './${filename}';`)) {
+		return `ERR: \`${filename}\` should be imported by \`${correspondingTsxFile}\``;
```

Comment:
Successful failure due to https://github.com/refined-github/refined-github/pull/6879#discussion_r1318472099

```
ERR: `link-to-changelog-file.gql` should be imported by `source/features/link-to-changelog-file.tsx`
```